server/src/main/java/org/eclipse/openvsx/LocalRegistryService.java

```diff
@@ -562,7 +562,7 @@ private SearchEntryJson toSearchEntry(ExtensionSearch searchItem, String serverU
         return json;
     }
 
-    private ExtensionJson toJson(ExtensionVersion extVersion) {
+    public ExtensionJson toExtensionVersionJson(ExtensionVersion extVersion) {
```

Comment:
See comment above: it's not necessary to make this public if you just use the built-in `toExtensionJson()` method of `ExtensionVersion`.